### ISSUE TITLE
Order of Building, Flashing, Running the Program

### ISSUE THREAD

**Issue by Mirabellensaft** [ferrous-systems/embedded-trainings-2020#88]
I kind of skipped the flashing part, does it make sense to talk about what flashing is, that it happens as part of running it out of VS Code, but leave out the command, for simplicity?

**Comment by Lotterleben** [comment]
yes! there were several points of confusion where people weren't sure if the code was running on the board or what was happening.
